src/app/shared/dialog-box/dialog-box.component.html

```diff
@@ -0,0 +1,24 @@
+<div mat-dialog-content>
+  <h2 class="govuk-heading-l">{{ data.context }}</h2>
+  <textarea
+    class="govuk-textarea govuk-!-width-three-quarters"
+    id="reasonForCreation"
```

Comment:
just checking whether this needs to be as test-reasonForCreation ? 
In case this goes to QA and we need to add it. If not we can resolve

Reply:
I think Umer is going to extract and change the modals so maybe leave it for him to add?

Reply:
if it is used by the automation tester this needs to be test-reasonForCreation as this is how they use the xpath selectors with the webdrivers. So if it is used by them this needs to be test-reasonForCreation otherwise their tests will fail. Best double checking with Dragos for this id to be sure.

Reply:
checked with Dragos, it needs to be test-reasonForCreation let's change it please so all the e2e tests pass